src/test/java/org/gridsuite/modification/server/modifications/VoltageLevelCreationTest.java

```diff
@@ -0,0 +1,105 @@
+package org.gridsuite.modification.server.modifications;
```

Comment:
copyright header is missing in this new file

Reply:
done